tests/bgp/test_bgp_dual_asn.py

```diff
@@ -0,0 +1,509 @@
+import pytest
+import time
+import logging
+import ipaddress
+import random
+import re
+
+from tests.common import constants
+from tests.common import config_reload
+from tests.common.utilities import wait_tcp_connection
+from tests.common.utilities import wait_until
+from tests.common.helpers.assertions import pytest_assert
+from tests.bgp.bgp_helpers import update_routes
+from tests.generic_config_updater.test_bgp_speaker import get_bgp_speaker_runningconfig
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
```

Comment:
GCU is only supported in 202205 and later branch. Need to bypass the test if build is earlier than 202205.

Reply:
Thanks for the information, will apply in latest diff.